psi4/src/read_options.cc

```diff
@@ -178,7 +178,7 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
     /*- What algorithm to use for the SCF computation. See Table :ref:`SCF
     Convergence & Algorithm <table:conv_scf>` for default algorithm for
     different calculation types. -*/
-    options.add_str("SCF_TYPE", "PK", "DIRECT DF MEM_DF DISK_DF PK OUT_OF_CORE CD GTFOCK COSX");
+    options.add_str("SCF_TYPE", "PK", "DIRECT DF MEM_DF DISK_DF PK OUT_OF_CORE CD GTFOCK COSX LINK");
```

Comment:
We don't have to do it this PR, as we have discussed in person; but eventually, we will want to add a framework for using delimiters and parsing CompositeJK J and K SCF_TYPES. Say, something that allows SCF_TYPES like "DIRECTDFJ+LINK" or "DIRECTDFJ+COSX" to be broken into their individual J+K components. This will be necessary as we add more separate J and K build algorithms to CompositeJK.